librz/debug/p/native/xnu/xnu_excthreads.c

```diff
@@ -47,7 +47,7 @@ RZ_IPI bool xnu_modify_trace_bit(RzDebug *dbg, xnu_thread_t *th, int enable) {
 #elif __POWERPC__ // ppc processor
 // XXX poor support at this stage i don't care so much. Once intel and arm done it could be done
```

Comment:
Could you please remove this comment while you are on it?

Reply:
I rephrased it